### PR TITLE
Renaming variable names in api-gateway microservice to agreed naming conventions (Depends on #285)

### PR DESCRIPTION
**Pull request by vishnusudarshan**
# Description

It's a PR for changing variable names to agreed naming conventions in api-gateway microservice. The PR should merge after #285  got merged.

## How Has This Been Tested

- Running integration test
- Running UI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api-gateway/src/rest/zkp.js [hunk 1]
```diff
@@ -290,7 +290,7 @@ export default {
   },
 
   // get fungible commitment token shield contract address for user address
-  getCoinShield({ address }) {
+  getFTokenCommitmentShield({ address }) {
```
Comment (iAmMichaelConnor) [inline api-gateway/src/rest/zkp.js]: I think maybe this should be getFTCommitmentShield, to be in line with the other name changes in this PR?

File: api-gateway/src/rest/zkp.js [hunk 2]
```diff
@@ -257,7 +257,7 @@ export default {
   },
 
   // get non-fungible commitment token shield address for user address
-  getTokenShield({ address }) {
+  getNFTokenCommitmentShield({ address }) {
```
Comment (iAmMichaelConnor) [inline api-gateway/src/rest/zkp.js]: I think maybe this should be getFTCommitmentShield, to be in line with the other name changes in this PR?

File: api-gateway/src/services/shield.js [hunk 3]
```diff
@@ -8,17 +8,17 @@ import { zkp } from '../rest';
 // eslint-disable-next-line import/prefer-default-export
 export async function getShieldAddresses(req, res, next) {
   try {
-    const coinShield = await zkp.getCoinShield(req.user);
-    const tokenShield = await zkp.getTokenShield(req.user);
+    const ftCommitmentShield = await zkp.getFTokenCommitmentShield(req.user);
+    const nftCommitmentShield = await zkp.getNFTokenCommitmentShield(req.user);
```
Comment (iAmMichaelConnor) [inline api-gateway/src/services/shield.js]: I think maybe this should be getFTCommitmentShield, to be in line with the other name changes in this PR?

File: api-gateway/src/rest/zkp.js (hunk 1 above)
Comment (vishnusudarshan) [reply api-gateway/src/rest/zkp.js]: Updated the variable name to getFTCommitmentShield

File: api-gateway/src/rest/zkp.js (hunk 2 above)
Comment (vishnusudarshan) [reply api-gateway/src/rest/zkp.js]: Updated the variable name to getNFTCommitmentShield

File: api-gateway/src/services/shield.js (hunk 3 above)
Comment (vishnusudarshan) [reply api-gateway/src/services/shield.js]: Updated the variable name to getNFTCommitmentShield

**Pull request merged**